### PR TITLE
Connects routes of results to details component and includes testing

### PR DESCRIPTION
**Pull request by priscillaapaxton**
# PR Template

### Types of changes made?

- [x]  testing
- [ ]  documentation
- [x]  javascript
- [x]  functionality

### What does this PR do?

- Adds functionality to routes within app to connect results component to details component
- Updates JS for results and card components including keys and proptypes
- Fixes bug for
- Writes test for app component

### Questions/Comments/Relevant Screenshots?
Link needs to be refactored within results component, app test needs to be refactored and additional css needs to be added to both components.

Screen Shot 2023-06-01 at 5 49 36 PM

### CODE REVIEW

File: src/Results/Results.js [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+import { Link } from 'react-router-dom';
+import './Results.css';
+
 
 const Results = ({places}) => {
-  return(places.map(place => {
+  const resultsCards = places.map(place => {
     return (
-      <div className='results-container'>
+      <Link key={place.id} to={`/details/${place.id}`}>
```
Comment (hjawad22) [inline src/Results/Results.js:11]: We can change this to a Navlink later down the line.

**Comment by hjawad22** [comment]
Good work on the first round of testing!

File: cypress/e2e/app_spec.cy.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+describe('App component', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+    cy.intercept('GET', 'http://localhost:8080/api/v1/places', {
```
Comment (tialaaa) [inline cypress/e2e/app_spec.cy.js:4]: Question for learning: This intercept is confirmed to actually be intercepting in Cypress? I had the impression that the `cy.visit` line should be after the `cy.intercept` but that could be incorrect.

File: src/Card/Card.js [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
-  )
-}
+  );
+};
 
-export default Card
+export default Card;
+
+Card.propTypes = {
+  src: PropTypes.string.isRequired,
```
Comment (tialaaa) [inline src/Card/Card.js:23]: propTypes look good!!

**Pull request merged**